tests/output_test.rs

```diff
@@ -0,0 +1,72 @@
+use std::{
+    env::temp_dir,
+    path::{Path, PathBuf},
+};
+
+use insta::sorted_redaction;
+use weighted_code_coverage::{GrcovFormat, Mode, WccRunner};
+
+const PROJECT_PATH: &str = "./tests/seahorse/";
+const SNAPSHOTS_PATH: &str = "./snapshots/output/";
+const COVERALLS_PATH: &str = "./tests/seahorse/coveralls.json";
+const COVDIR_PATH: &str = "./tests/seahorse/covdir.json";
+const JSON_OUTPUT: &str = "wcc.json";
```

Comment:
Can we use `concat!` to join these constants and then pass them as `Path` instead of `PathBuf`?